--- nautilus-3.5.92/src/Makefile.in.orig	2012-09-18 08:32:53.000000000 +0800
+++ nautilus-3.5.92/src/Makefile.in	2012-09-18 09:35:20.000000000 +0800
@@ -491,6 +491,7 @@
 	$(EXIF_LIBS) \
 	$(EXEMPI_LIBS) \
 	$(POPT_LIBS) \
+	-lgmodule-2.0 \
 	$(NULL)
 
 dbus_freedesktop_built_sources = \
--- nautilus-3.5.92/test/Makefile.in.orig	2012-09-18 08:32:53.000000000 +0800
+++ nautilus-3.5.92/test/Makefile.in	2012-09-18 09:35:20.000000000 +0800
@@ -358,6 +358,7 @@
 	$(BASE_LIBS) \
 	$(COMMON_LIBS) \
 	$(NAUTILUS_LIBS) \
+	-lgmodule-2.0 \
 	$(NULL)
 
 test_nautilus_copy_SOURCES = test-copy.c test.c
